### PR TITLE
Italian translation added

### PR DESCRIPTION
**Pull request by enricolucia**
I've added an Italian translation of the Readme.
Feel free to include that in the repo if you wish.

Thanks

### CODE REVIEW

File: README-it.md [hunk 1]
```diff
@@ -0,0 +1,650 @@
...
+
+Il mondo del Frontend si evolve rapidamente oggigiorno, i browsers moderni hanno gia' implementato un'ampia gamma di DOM/BOM API soddisfacenti. Non dobbiamo imparare jQuery dalle fondamenta per la manipolazione del DOM o di eventi. Nel frattempo, grazie al prevalicare di librerie per il frontend come React, Angular a Vue, manipolare il DOM direttamente diventa un anti-pattern, di consequenza jQuery non e' mai stato meno importante. Questo progetto sommarizza la maggior parte dei metodi e implementazioni alternative a jQuery, con il supporto di IE 10+.
+
+## Tabella contenuti
+
+1. [Query Selector](#query-selector)
+1. [CSS & Style](#css--style)
+1. [Manipolazione DOM](#dom-manipulation)
+1. [Ajax](#ajax)
+1. [Eventu](#events)
```
Comment (camsong) [inline README-it.md]: link broken

**Comment by camsong** [comment]
@enricolucia Thanks for the contribution :clap: really like it.
You can also add a link in the https://github.com/oneuijs/You-Dont-Need-jQuery#translations section.

**Comment by enricolucia** [comment]
@camsong you are welcome.
Just added in there.

File: README-it.md (hunk 1 above)
Comment (enricolucia) [inline README-it.md]: Where is the broken link?

**Comment by camsong** [comment]
Can you also fix other broken links.

Can you squash these commits into one? May refer to https://davidwalsh.name/squash-commits-git

If you don't know how to squash, the easy way is to save your changes first -> delete your fork and fork from here again -> apply your changes manually -> create a new PR.

File: README-it.md [hunk 2]
```diff
@@ -0,0 +1,650 @@
...
+
+1. [Query Selector](#query-selector)
+1. [CSS & Style](#css--style)
+1. [Manipolazione DOM](#dom-manipulation)
+1. [Ajax](#ajax)
+1. [Eventi](#events)
+1. [Utilities](#utilities)
+1. [Alternative](#alternatives)
+1. [Traduzioni](#translation)
+1. [Supporto Browsers](#browser-support)
```
Comment (camsong) [inline README-it.md]: those links also broken, please help to check others.

Comment (enricolucia) [inline README-it.md]: sure I am checking them now.

**Comment by enricolucia** [comment]
Just squashed and pushed.

**Comment by camsong** [comment]
:+1: great

**Comment by enricolucia** [comment]
Pleased to contribute :+1:

**Pull request merged**